sdk/modelsrepository/Azure.Iot.ModelsRepository/src/RepositoryHandler.cs

```diff
@@ -16,19 +17,22 @@ internal class RepositoryHandler
         private readonly IModelFetcher _modelFetcher;
         private readonly Guid _clientId;
         private readonly ClientDiagnostics _clientDiagnostics;
+        private readonly Uri _repositoryUri;
+        private readonly ResolverClientOptions _clientOptions;
 
-        public Uri RepositoryUri { get; }
-        public ResolverClientOptions ClientOptions { get; }
-
-        public RepositoryHandler(Uri repositoryUri, ClientDiagnostics clientdiagnostics, ResolverClientOptions options = null)
+        public RepositoryHandler(Uri repositoryUri, ClientDiagnostics clientdiagnostics, ResolverClientOptions options)
```

Comment:
`clientDiagnostics` (casing D)

Also, why is this a distinct parameter and not in client options?

Reply:
That is a good question, I haven't seen any other client have the diagnostics as part of the options

Reply:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/samples/Diagnostics.md